react-validated-input-component/my-validated-input-component/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import { FaTimes, FaCheck } from 'react-icons/fa';
+
+export default function ValidatedInput() {
+  const [isValid, setIsValid] = useState('');
```

Comment:
It looks like this isn't really a boolean that indicates if the password is valid. It's the actual password the user typed! Maybe it should be renamed....